src/main/java/edu/kis/powp/jobs2d/drivers/adapter/TransformDriverAdapter.java

```diff
@@ -0,0 +1,22 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+
+
+public class TransformDriverAdapter extends LineDriverAdapter {
+
+	public TransformDriverAdapter(DrawPanelController drawController, ILine line, String name) {
+		super(drawController, line, name);
+	}
```

Comment:
Transformation should not be added simply to a single driver. This should be a generic solution. Try e.g. decorator pattern.